### ISSUE TITLE
Problem with scrolling the list for a flex container

### ISSUE THREAD

**Issue by OPITOs** [GeekyAnts/NativeBase#3862]
[https://snack.expo.dev/I9rb3P83X](https://snack.expo.dev/I9rb3P83X)

```
import React from "react";
import {
  View,
  Text,NativeBaseProvider
} from "native-base";
import {FlatList} from 'react-native'

export default function () {
  return (
    <NativeBaseProvider>
      <FlatList 
        data={new Array(100).fill(0)} 
        renderItem={()=>
          <View flex={1}>
            <Text>XXXXXXXXXXXXXXXX</Text>
          </View>
        }
      />
    </NativeBaseProvider>
  );
}
```

Problem for ios and android.
Accident list for a container with flex. Converting flex to flexGrow works fine.

**Comment by amars29** [comment]
@OPITOs Please update the native-base version to 3.2.0 or above.

**Comment by surajahmed** [comment]
This issue has been fixed in version `3.2.0` or later.
